### PR TITLE
[docs] add info about ci

### PR DESCRIPTION
**Pull request by dmlemeshko**
## Summary

This PR adds info about how to run performance testing on Kibana CI: links, available parameters, where to get results.

### Checklist

Delete any items that are not applicable to this PR.

- [ ] Branch is successfully tested on [Kibana CI](https://kibana-ci.elastic.co/job/elastic+kibana+load-testing/)
- [ ] Unit tests are added

### CODE REVIEW

File: docs/KIBANA_CI.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Start a new run:
+- go to [load-testing](https://kibana-ci.elastic.co/view/Kibana/job/elastic+kibana+load-testing/) job
+- click 'Build with Parameters'
+- enter your github username & branch name, simulation classes, uncheck 'INGEST RESULTS' if you don't need it in Kibana stats  
+- click 'Build'
+![branch testing job](img/Jenkins_branch_testing.png)
+
+Process:
+- build Kibana from branch source code
+- start Kibana and ES using FTR
```
Comment (rashmivkulkarni) [inline docs/KIBANA_CI.md:17]: probably you could add how to do it here or link it to the doc which tells how to.

Comment (LeeDr) [reply docs/KIBANA_CI.md:17]: Rashmi, I think the `Start a new run:` are the steps how.  The `Process:` part is saying what the job does.  Not what you have to do.

Comment (dmlemeshko) [reply docs/KIBANA_CI.md:18]: Addressed in f3804401f36d776e66cf99285e5b44a83d9877f7
Should be clear now :)

Comment (rashmivkulkarni) [reply docs/KIBANA_CI.md:18]: more clear now . Thanks Dima

**Review by LeeDr — APPROVED** [review]
LGTM

**Pull request merged**